### ISSUE TITLE
Module 3, Exercise 1, Step 14  -need to disable soft delete for Blob service in Gen Purpose Storage Acct. v2

### ISSUE THREAD

**Issue by LionelShaul** [MicrosoftLearning/AZ-304-Microsoft-Azure-Architect-Design#106]
# Module: 03
#### Step: 14

Description of issue
Cannot initiate replication of VM if Soft Delete is enabled for Blob service
Repro steps:

1. Error will occur in Exercise 3, Task 2, Step 7.
2.Viewing the Notification message will show Replication failed.  
Error detail will state:
Error ID
28177
Error Message
Soft delete is enabled for the storage account 'inisa304' provided.
Possible causes
Soft delete enabled storage account is not supported.
Recommendation
Go to the storage account and open ‘Data protection’ settings. Disable soft delete for blobs. Or, provide a storage account which has Soft Delete disabled. Resolve the issue and retry the operation. If the problem persists, contact support.
3. Disabling Soft Delete remedies the problem.

**Comment by wchandra** [comment]
Agreed. I faced the same issue and it was resolved after the Storage account setting was updated as recommended.
